### PR TITLE
Added Netskope Solution

### PR DESCRIPTION
**Pull request by nipun-crestdatasystem**
Change(s):
   - Added Updated Netskope solution

   Reason for Change(s):
   - Added Updated Solution

   Version Updated:
   - 3.0.0

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Solutions/Netskope/Parsers/Netskope.yaml
Comment (v-prasadboke) [inline Solutions/Netskope/Parsers/Netskope.yaml:1]: Any reason behind removing Netskope.yaml

File: Solutions/Netskope/Package/2.0.0.zip
Comment (v-prasadboke) [inline Solutions/Netskope/Package/2.0.0.zip:1]: Please add this removed zip packages

Comment (nipun-crestdatasystem) [reply Solutions/Netskope/Package/2.0.0.zip:1]: Our client wants to remove the older data connector hence we have removed it. we still need to provide or add those zip packages ?

**Comment by v-prasadboke** [comment]
Hello @nipun-crestdatasystem, 
Please work on the above mentioned changes.

**Comment by nipun-crestdatasystem** [comment]
We have raised the new PR with updated change by adding new folder named Netskopev2 instead of replacing the existing one. Hence closing this PR.

**Pull request closed without merging**